src/app/lib/getAccountBalanceWithFallback.ts

```diff
@@ -0,0 +1,36 @@
+import { addressToPublicKey, parseRpcBalance } from 'app/lib/helpers'
+import { call } from 'typed-redux-saga'
+import { getExplorerAPIs, getOasisNic } from '../state/network/saga'
+import { Account } from '../state/account/types'
+
+function* getBalanceGRPC(address: string) {
+  const nic = yield* call(getOasisNic)
+  const publicKey = yield* call(addressToPublicKey, address)
+  const account = yield* call([nic, nic.stakingAccount], { owner: publicKey, height: 0 })
+  const grpcBalance = parseRpcBalance(account)
+  return {
+    address,
+    available: grpcBalance.available,
+    delegations: null,
+    debonding: null,
+    total: null,
```

Comment:
> @csillag: As far as I can tell, currently when the API call fails, and we fall back to the gRPC, this is all done silently. So just by looking at the UI, there is no way to tell whether we are seeing the real total balance, or just the portion returned by the gRPC fallback.

This fallback returns null total, rather than `available + 0 + 0` so the displayed balances shouldn't be misleading. But I noticed we don't differentiate loading and null. It just keeps loading
![image](https://github.com/oasisprotocol/oasis-wallet-web/assets/3758846/8d3aabd1-ec8d-4857-9623-85c9b55a5d83)

Reply:
Fixed 
![image](https://github.com/oasisprotocol/oasis-wallet-web/assets/3758846/ca3c6a00-06de-4f6c-8b46-15b035877006)